pkg/controller/perconaservermongodb/finalizers.go

```diff
@@ -120,7 +120,7 @@ func (r *ReconcilePerconaServerMongoDB) deletePSMDBPods(ctx context.Context, cr
 			rsDeleted = false
 			switch err {
 			case errWaitingTermination, errWaitingFirstPrimary:
-				log.Error(err, "rs", rs.Name)
+				log.Info("deleting rs pods", "rs", rs.Name, "status", err.Error())
```

Comment:
Why we use info not error here?

Reply:
because it's not actually an error and it's confusing. request came from @eleo007